source/draw/Color.ooc

```diff
@@ -23,7 +23,9 @@ ColorMonochrome: cover {
 	toRgb: func -> ColorRgb { this toYuv() toRgb() }
 	toRgba: func -> ColorRgba { this toYuv() toRgba() }
 	equals: func (other: This) -> Bool { this y == other y }
-	blend: func (factor: Float, other: This) -> This { This new((this y * (1 - factor) + (other y * factor)) as Byte) }
+	mix: static func (a, b: This, ratio: Float) -> This {
+		This new(Float mix(a y, b y, ratio) as Byte)
+	}
```

Comment:
Now conforming to GLSL naming that works for both vectors and colors. The argument order conforms to both HLSL and GLSL.